run_container.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e 
+
+PORT=3490
```

Comment:
I would change slightly here to avoid trying to have some issues with two containers trying to access the same port
```bash
if [[ $(docker ps | grep tester) ]]; then
    docker kill tester
fi

docker build -t webserv .
docker run -dp $PORT:$PORT webserv --name tester
```

Reply:
Could you explain a bit please? What is this tester program in docker ps?

Reply:
A Docker container is a process - with a randomly assigned PID. with the tag `--name` you should be able to give it a name and therefore being able to kill the process (the docker container) easily by launching the command `docker kill <chosen-name>`

Here a screenshot to demonstrate:
![image](https://user-images.githubusercontent.com/36924670/214378481-56c28413-3e09-4d4f-99e7-daaa06ab2ba6.png)

Reply:
Ok thanks for explaining! made the edit!

Reply:
you missed the `--name` tag - I have added a commit to fix that